fiber-unix/src/worker.mli

```diff
@@ -0,0 +1,19 @@
+(** Simple queue that is consumed by its own thread *)
+type 'work t
+
+(** [create do_] creates a worker with a task handler [do_]. The worker will not
+    handle an exception raised by the task handler, so [do_] is expected to not
+    raise. *)
+val create : do_:('a -> unit) -> 'a t
```

Comment:
We could rename the label to something that reminds of the limitation. For example: `do_no_raise`.